added scenarios AUTH_01, AUTH_02, PM_01

yurii-b-gh opened the pull request:
added:
UML diagram for scenario AUTH_01
UML diagram for scenario AUTH_02
UML diagram for scenario PM_01

yurii-b-gh committed:
added scenario AUTH_01

added:
UML diagram

yurii-b-gh committed:
added scenario for AUTH_02

added UML diagram for scenario AUTH_02

yurii-b-gh committed:
Виправлення синтаксичних помилок

yurii-b-gh committed:
added scenario PM_01

gurug-prog reviewed (changes requested):
Good, but there are some issues.

General:
- you don't need to add the diagram code to the docs page

AUTH_01, AUTH_02:
There some processes before the user opened auth or register form kinda:
- user pressed an auth button
- the system gives an auth form to user
- user is filling the form
- ...

Also in AUTH_02 you have a typo with exceptions numbering (should be AUTH_EX_02.01, AUTH_EX_02.02).

PM_01:
- the same problem with creating project button as in the auth/register form
- exceptions cannot happen on user side, they are always thrown by the system

yurii-b-gh committed:
Changes in diagrams

changes:
- exceptions in scenario AUTH_02 changed
- added more details in scenarios
- fixed mistakes

gurug-prog reviewed (commented):
LGTM!

gurug-prog committed:
Merge branch 'master' into yurii

socheretenko commented:
LGTM!
